lib/src/blocs/authentication_blocs/authentication_blocs.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter_bloc/flutter_bloc.dart';
+import '/src/blocs/authentication_blocs/authentication_event.dart';
+import '/src/repositories/user_repository.dart';
```

Comment:
/src/blocs/authentication_blocs/authentication_event.dart 
=> ../blocs/authentication_blocs/authentication_event.dart

Reply:
I can not naming this folder like this 
![image](https://user-images.githubusercontent.com/33689489/127881709-90ec6a56-cf63-430f-b54e-579aac437426.png)

Reply:
../authentication_event.dart hope this help

Reply:
![image](https://user-images.githubusercontent.com/86453578/127950720-684d9bd7-b497-40df-84ba-659c11914205.png)